heureka/ui/src/App.js

```diff
@@ -27,6 +34,33 @@ const App = (props) => {
   w-full
   `
 
+  useLayoutEffect(() => {
+    // filterLabels are the labels shown in the filter dropdown, enabling users to filter services based on specific criteria. Default is support-group.
+    if (props.filterLabels) setLabels(props.filterLabels)
+
+    // predefined filters config
+    const predefinedFilters = [
+      {
+        name: "support-group",
+        displayName: "support-group",
+        // regex that matches all entities with default filter
+        region: "\bcontainers\b",
+      },
+    ]
+    // setPredefinedFilters(predefinedFilters)
+
+    // initially active predefined filter
+    const initialPredefinedFilter = "support-group"
```

Comment:
This is fine for now but you should make this configurable eventually.

Reply:
I will set the predefined filter of support group in a separate PR.